caniusepython3/__main__.py

```diff
@@ -118,12 +126,12 @@ def pprint_blockers(blockers):
     return pprinted
 
 
-def check(projects):
+def check(projects, index_url):
```

Comment:
If we're going to import `pypi` here then might as well provide a default argument.